lib/linux_ccnt/aarch64_cycle_counter.c

```diff
@@ -0,0 +1,53 @@
+#include <linux/kernel.h>
```

Comment:
Please add copyright information for all new files.

Reply:
Generated the file with copyright info